lsp-mode.el

```diff
@@ -8385,6 +8385,43 @@ This avoids overloading the server with many files when starting Emacs."
     (lsp--error "Nothing to disconnect from?")))
 
 
+;;; `lsp-iedit'
```

Comment:
Can we move this to a separate file? otherwise `lsp-mode.el`  will explode 😅

Reply:
Sure.